### PR TITLE
Update matchms to 0.18.0

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`matchms`](https://bioconda.github.io/recipes/matchms/README.html): **0.17.0** &rarr; **0.18.0**

[install with bioconda](http://bioconda.github.io/recipes/matchms/README.html) [Conda](https://anaconda.org/bioconda/matchms/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/matchms`](https://github.com//bioconda/bioconda-recipes/tree/bump/matchms/recipes/matchms) (click to view/edit other files)
Summary | Python library for fuzzy comparison of mass spectrum data and other Python objects
Home | [https://github.com/matchms/matchms](https://github.com/matchms/matchms)
Releases |[https://pypi.org/pypi/matchms/json](https://pypi.org/pypi/matchms/json)
Recipe Maintainer(s) | @hechth, @maximskorik

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by florian-huber** [comment]
Dependencies need to be updated:
- add `sparsestack >= 0.4.1`
- change pickydict to `pickydict >= 0.4.0`

File: recipes/matchms/meta.yaml [hunk 1]
```diff
@@ -30,7 +30,9 @@ requirements:
     - rdkit >=2020.03.1
     - requests
     - scipy >=1.4.0
-    - pickydict >=0.3.0
+    - sparsestack >=0.4.1
+    - pickydict >=0.4.0
+    - pyyaml
```
Comment (hechth) [inline recipes/matchms/meta.yaml:35]:
```suggestion
    - sparsestack >=0.4.1
    - pickydict >=0.4.0
    - pyyaml
    - tqdm
```

**Review by hechth — COMMENTED** [review]
tqdm should be added to the dependencies as well

File: recipes/matchms/meta.yaml (hunk 1 above)
Comment (florian-huber) [reply recipes/matchms/meta.yaml:35]: Since I have no editing rights I added a new PR for this #38675 .

**Review by hechth — APPROVED** [review]

**Review by hechth — APPROVED** [review]

**Comment by hechth** [comment]
@BiocondaBot please add label

**Review by florian-huber — APPROVED** [review]

**Comment by maximskorik** [comment]
@bioconda/core hi there! Could anyone take a look, please?

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Pull request merged**